docs/kb/integration/defect-dojo-integration.md

```diff
@@ -0,0 +1,114 @@
+---
+tags:
+  - DefectDojo
+  - Integration
+description: How to connect Semgrep and DefectDojo
+---
+
+# How to connect Semgrep and DefectDojo
+
+[DefectDojo](https://www.defectdojo.com/) is a well-known vulnerability management tool. It allows you to gather security issues from tools like Semgrep and more. By integrating Semgrep findings into DefectDojo, security teams gain additional insight into the security posture of their systems.
+
+## Starting with integration
+Integrating Semgrep with DefectDojo means dumping findings detected by Semgrep into DefectDojo.
+There are several steps to follow:
+
+1. In DefectDojo:
+    1 Create your **product**. A [product](https://defectdojo-dev.readthedocs.io/en/latest/features.html#products) is DefectDojo's term for a project.
```

Comment:
"... is DefectDojo's term for a project."

A project of what? Do you mean a project like Jira or like GitHub's Projects feature?

(You can also remove the line.)